CHANGELOG.md

```diff
@@ -2,22 +2,30 @@
 
 This changelog follows the following convention [https://keepachangelog.com/en/1.0.0/](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.2] - 2019-09-??
 
 ### Breaking
 
 - `textworld.envs.wrappers.Filter` expects the environment to wrap as its first argument.
 - `textworld.logic.State` now requires the `GameLogic` to be provided, so that it can know about the type hierarchy of each variable.
 
-### Fixed
+### Removed
 
-- `tw-make tw-coin_collector` was failing with option `--level {100, 200, or 300}`.
-- Quest tracking was failing when an irreversible, but unneeded, action was performed.
+- `textworld.envs.FroztEnv` (use `textworld.envs.JerichoEnv` instead)
```

Comment:
`Frotz`?

Reply:
Actually, the environment was called `FrotzEnvironment`. That's your question?

Reply:
I mean instead of `Frozt`

Reply:
:laughing: yes oops.